app/src/main/java/org/mozilla/rocket/content/travel/domain/GetCityWikiUseCase.kt

```diff
@@ -24,12 +24,12 @@ fun String.trimContentWithinParentheses(): String {
 
     val trimmed = this.filter {
 
-        if (it.equals('(')) {
+        if (it.equals('(') || it.equals('（')) {
```

Comment:
`it.equals('（')` and others can be simplified to `it == '（'`. It's not a merge blocker.